share/spice/isslocation/isslocation.js

```diff
@@ -0,0 +1,38 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_isslocation = function(api_result) {
+        
+       if(!api_result || api_result !== "success") {
```

Comment:
@javathunderman probably best to check
```js
if (!api_result || !api_result.message || api_result.message !== "success"){
```

If we know the message is always there, you can leave out the check for the existance of `api_result.message`